docs/router/plugins.mdx

```diff
@@ -0,0 +1,265 @@
+  </Card>
+
+  <Card title="Design with GraphQL, Implement with gRPC" icon="code">
+    Our approach combines GraphQL's schema-first flexibility with gRPC's performance and reliability. Using the same plugin system that powers HashiCorp Vault and Terraform, you can develop subgraphs in any language and run them directly in the Cosmo Router.
+  </Card>
+
+  <Card title="Proto-Based Code Generation" icon="bolt">
+    The strict typing and automatic code generation from Protocol Buffers (proto) definitions provides an immense productivity boost, especially in the era of generative AI. The generated proto-based gRPC code creates a strongly-typed foundation that AI tools can more effectively understand, extend, and modify, reducing development time and errors while ensuring consistent API implementation.
+  </Card>
+</CardGroup>
+
+### Benefits Over Traditional Subgraphs
+
+When compared to traditional standalone subgraphs, gRPC plugins offer substantial technical advantages:
+
+<CardGroup cols={2}>
+  <Card title="Reduced Latency" icon="gauge-high">
+    Eliminates network round-trips between the router and subgraph for faster query execution
+  </Card>
+  <Card title="Simplified Deployment" icon="cloud-arrow-up">
+    Fewer moving parts means simpler deployments and maintenance with all subgraphs running in the router process
+  </Card>
+  <Card title="Reduced Infrastructure Costs" icon="dollar-sign">
+    Consolidate services to reduce overall resource consumption and operational complexity
+  </Card>
+  <Card title="Direct API Bridging" icon="bridge">
+    Easily connect legacy REST APIs, databases, and other services to your graph without building standalone GraphQL services
+  </Card>
+</CardGroup>
+
+This architecture gives you a unified development experience with consistent tooling across all your subgraphs while enabling seamless integration with your existing infrastructure.
+
+## How It Works
+
+The gRPC plugin architecture works by:
+
+1. **Plugin Registration**: Plugins are registered with the Cosmo Router at startup
+2. **Schema Integration**: The plugin's schema is integrated into the federated graph
+3. **Request Handling**: When the router receives a GraphQL request, it routes relevant parts to the appropriate plugins
+4. **Inter-Process Communication**: Plugins run in separate processes and communicate with the router via gRPC
+5. **Response Assembly**: The router assembles the complete response from all plugins and regular subgraphs
+
+## Use Cases
+
+gRPC plugins are ideal for:
+
+- **Legacy API Integration**: Wrap existing REST, SOAP, or other APIs behind GraphQL types
+- **Performance-Critical Paths**: Implement frequently accessed resolvers with minimal overhead
+- **Simplified Architecture**: Reduce the number of services in your deployment
+- **Resource Optimization**: Reduce cloud costs by consolidating services
```

Comment:
This duplicates the cards above, point for point

Reply:
Good point, I also had a hard time to distinguish them.